tests/phpunit.xml

```diff
@@ -1,13 +1,19 @@
-<phpunit bootstrap="../vendor/autoload.php" colors="true">
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/6.2/phpunit.xsd"
+         bootstrap="../vendor/autoload.php" 
+         colors="true">
     <testsuites>
         <testsuite name="DAMADoctrineTestBundle test suite">
-            <directory suffix="Test.php">./src</directory>
+            <directory>../tests</directory>
         </testsuite>
     </testsuites>
 
     <filter>
         <whitelist processUncoveredFilesFromWhitelist="true">
-            <directory suffix=".php">src</directory>
+            <directory suffix=".php">../src</directory>
+            <exclude>
+                <file>../src/DAMA/DoctrineTestBundle/PHPUnit/LegacyPHPUnitListener.php</file>
```

Comment:
can we also exclude the non legacy listener file?

Reply:
It's not needed, the legacy needs to be excluded or the PHPUnit bin will fatal; but the issue is present only if you need to run the coverage, so you need to exclude the non-legacy one only if you run it on PHP 5.6, where PHPUnit 6 is not usable.

Reply:
Ok I see :+1: